sample-docker.env

```diff
@@ -0,0 +1,32 @@
+MONGOHQ_URL=mongodb://mongo:27017/freecodecamp
```

Comment:
The first change is because it uses `'` as part of the url (something like `%27mongodb://localhost:27017/freecodecamp%27` I don't know why). The second one is to use the link to the `mongo` container.

Should I remove this file?